### PR TITLE
Rewrite tests not to use `numpy_cupy_raises` cupyx.* tests

### PR DESCRIPTION
**Pull request by asi1024**
Ready for #3098.

### CODE REVIEW

File: tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py [hunk 1]
```diff
@@ -747,36 +778,40 @@ def test_rmul_dense_matrix(self, xp, sp):
...
+            with pytest.raises(ValueError):
+                x * m
+
+    def test_rmul_unsupported(self):
+        for xp, sp in ((numpy, scipy.sparse), (cupy, sparse)):
+            m = self.make(xp, sp, self.dtype)
+            # TODO(unno): When a sparse matrix has no element, scipy.sparse
+            # does not raise an error.
+            if m.nnz == 0:
+                raise pytest.skip()
```
Comment (niboshi) [inline tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py]:
```suggestion
                continue
```

File: tests/cupyx_tests/scipy_tests/sparse_tests/test_csr.py [hunk 2]
```diff
@@ -788,41 +820,46 @@ def test_rmul_dense_matrix(self, xp, sp):
...
-        None * m
+    def test_rmul_unsupported(self):
+        for xp, sp in ((numpy, scipy.sparse), (cupy, sparse)):
+            m = self.make(xp, sp, self.dtype)
+            if m.nnz == 0:
+                # When there is no element, a SciPy's sparse matrix does
+                # not raise an error when it is multiplied with None.
+                pytest.skip()
+                with pytest.raises(TypeError):
+                    None * m
```
Comment (niboshi) [inline tests/cupyx_tests/scipy_tests/sparse_tests/test_csr.py]: Use `continue` and fix indentation.

**Comment by asi1024** [comment]
@niboshi PTAL.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by toslunar** [comment]
`scipy_name='scp'` elsewhere.

**Pull request merged**